src/lib/swapper/swappers/ThorchainSwapper/utils/getLongtailQuote.ts

```diff
@@ -138,6 +138,9 @@ export const getLongtailToL1Quote = async (
         allowanceContract: ALLOWANCE_CONTRACT,
       })),
       isLongtail: true,
+      longtailData: {
+        longtailToLExpectedAmountOut: quotedAmountOut,
```

Comment:
And here too

Reply:
https://github.com/shapeshift/web/pull/6052/commits/436e8b7537e640938e5900bcec78221090702f70